src/opensearch-dashboards/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Use OpenSearch image as base
+FROM opensearchproject/opensearch-dashboards:2.7.0
+
+# Install plugin (https://opensearch.org/docs/1.2/dashboards/install/plugins/)
+USER root
+RUN sudo /usr/share/opensearch-dashboards/bin/opensearch-dashboards-plugin install <plugin-url>
```

Comment:
would users have to manually build plugin, upload to some server, update `<plugin-url>`? what if users don't need to change plugin from official image?

Also if plugin exists, should uninstall first then install

Reply:
users than need changes to the OpenSearch-dashboards can use this template
https://opensearch.org/docs/1.2/dashboards/install/plugins/#install - same api for create/update plugin